functions/package.json

```diff
@@ -17,9 +17,11 @@
   "main": "lib/functions/src/index.js",
   "dependencies": {
     "@actnowcoalition/assert": "^0.1.1",
+    "@types/node-fetch": "2.6.2",
     "firebase": "^9.10.0",
     "firebase-admin": "^10.0.2",
     "firebase-functions": "^3.24.1",
+    "node-fetch": "2.6.2",
```

Comment:
Using `node-fetch` v3 causes issues with Jest (see https://github.com/node-fetch/node-fetch/issues/1289) so I'm intentionally pinning this to 2.6 for the time being